src/Files.App/Services/Settings/IAppearanceSettingsService.cs

```diff
@@ -53,5 +56,30 @@ public interface IAppearanceSettingsService : IBaseSettingsService, INotifyPrope
 		/// Gets or sets a value for the theme system backdrop.
 		/// </summary>
 		BackdropMaterialType AppThemeBackdropMaterial { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value for the app background image source
+		/// </summary>
+		string AppBackgroundImageSource { get; set; }
```

Comment:
Following others,
AppThemeBackgroundImageSource sounds good.

Others below too.

Reply:
Resolved in the latest commit